commenting_system/migrations/0002_test_data.py

```diff
@@ -26,15 +26,15 @@ def generate_data(apps, schema_editor):
         Post_list = [
             Post(
                 nameOfPoster='Leead',
-                nameOfLocation='Eilat',
-                photoURL='test_1.com',
+                nameOfLocation='Sea of Galilee',
+                photoURL="https://www.shappo.co.il/Resize_Image.aspx?maxsize=400&img=/pictures/cards/big/36630.jpg",
                 Description='Perfect!',
             ),
             Post(
                 nameOfPoster='Shoval',
-                nameOfLocation='Tel-Aviv',
-                photoURL='test_2.com',
-                Description='This is nice',
+                nameOfLocation="`En Yorqe`am",
+                photoURL="https://cdn1.sipurderech.co.il/1200x800_fit_90/1403218722_121.jpeg",
```

Comment:
this either is not working for me 
![image](https://user-images.githubusercontent.com/48455448/116584854-73b19e80-a920-11eb-9e06-774f5e7f4dda.png)

Reply:
Did you follow the steps?
You need to delete the DB file.